### DISCUSSION TITLE
Create portable lists of folders / apps to be uninstalled?

### DISCUSSION THREAD

**Discussion by ghost** [Klocman/Bulk-Crap-Uninstaller#587] [Q&A]
Many moons ago I used PC Decrapifier which allowed my to make lists of files / folders / apps that could be moved from one machine to another and ultimately allowed me to build up a good-sized list of rubbish to remove from various machines.

Can BCU allow me to do this as well? My experimenting so far doesn't seem to show it can be done.

**Comment by Klocman** [comment]
Open advanced filters and make your list in there. Check the uninstall list section in the readme for more info.

**Reply by ghost** [reply]
Looked, read but have trouble working out what to do.

Are there any examples online anywhere?
